agent/engine/task_manager.go

```diff
@@ -623,7 +617,7 @@ func (mtask *managedTask) emitTaskEvent(task *apitask.Task, reason string) {
 		})
 		return
 	}
-	logger.Info("Sending task change event", logger.Fields{
+	logger.Debug("Sending task change event", logger.Fields{
```

Comment:
same - if logs from customers are just at info level, would we know if agent actually sent state change events to backend?

Reply:
For this one we will know. This log is a dupe of this one, which is the log right before sending data through the wire.

Reply:
should we just remove this log then?